src/handlers/list.ts

```diff
@@ -1,10 +1,28 @@
+import { readFile } from "node:fs/promises";
+import { Task } from "../models/Task";
+
 export namespace ListHandler {
   export type Args = {
     all: boolean;
   };
 
+  const taskData = "data/db.json";
```

Comment:
This is probably going to be used by other handlers, we may want to move it to somewhere else

Reply:
I agree! Do you mind if I tackle this on the next PR?

Reply:
No problem! Let's move it once it's needed!